### ISSUE TITLE
Cannot create patient resources (POST, PUT) when deploying r4-fhir-server:latest docker image to Azure as App service

### ISSUE THREAD

**Issue by marincon-sarmiento** [microsoft/fhir-server#2927]
**Question**

I am having trouble creating patients using POST and PUT HTTP calls I will provide a sample of the calls in the description section.  

**Description**
I deployed the docker-compose.yaml provided [here](https://github.com/microsoft/fhir-server/tree/main/samples/docker) locally on my laptop. That is working without problem. I am able to create patients using POST (http://<BaseUrl>/Patient) and PUT (http://<BaseUrl>/Patient/SomeUniqueId) calls with the following body:
`{
    "resourceType": "Patient",
    "id": "24c32135-d0af-41e1-acfb-ac6a80c17471",
    "identifier": [
        {
            "use": "usual",
            "type": {
                "text": "Patient unique identifier in NUPA system."
            },
            "system": "company.com",
            "value": "24c32135-d0af-41e1-acfb-ac6a80c17471"
        },
        {
            "use": "official",
            "type": {
                "text": "Patient national identifier."
            },
            "system": "government.ca",
            "value": "secondIdentifier"
        }
    ],
    "active": true,
    "name": [
        {
            "use": "usual",
            "text": "Pepito Peres Rodriguez Rincon",
            "family": "Rodriguez Rincon",
            "given": "Pepito Peres",
            "prefix": "",
            "suffix": ""
        }
    ],
    "telecom": [
        {
            "system": "email",
            "value": "pepito@email.com",
            "use": "mobile",
            "rank": 1
        }
    ],
    "gender": "male",
    "birthDate": "2003-04-08",
    "deceasedBoolean": false,
    "address": [
        {
            "use": "home",
            "type": "both",
            "text": "135 Walabee St",
            "line": [
                "135",
                "Walabee St"
            ],
            "city": "Montreal",
            "district": "",
            "state": "Quebec",
            "postalCode": "A1A 1A1",
            "country": "CA"
        }
    ]
}`

Locally I always get 201 status calls, so the patient resource is being successfully created. I also have tried getting the patients both with through the list GET (http://<BaseUrl>/Patient) and their specific id GET (http://<BaseUrl>/Patient/SomeUniqueId) and those work as well. 

The problem is when I deploy the image to the cloud as an app service. Here are some more details:
* I deployed it manually to the cloud by creating a docker-based app service and referring to the docker hub image for the R4 FHIR server (mcr.microsoft.com/healthcareapis/r4-fhir-server:latest), however when I try to create patients using POST or PUT I get search results as the responses: 
POST (http://<AzureUrl>/Patient)
![image](https://user-images.githubusercontent.com/48895453/199570436-d9ce07b5-f483-4485-abb7-8939d9a23f48.png)

PUT (http://<AzureUrl>/Patient/SomeUniqueId)
![image](https://user-images.githubusercontent.com/48895453/199570754-bec5a15a-37c2-4f66-b783-63dde34ae272.png)

* The storage is provided by an Azure managed SQL database; the connection string is being accessed by the App service from a key vault in Azure. 
* I provided the image with the same environment variables as the ones on the docker-compose.yaml here (except for the connection string):
```
FHIRServer__Security__Enabled: "false"
SqlServer__AllowDatabaseCreation: "true"
SqlServer__Initialize: "true"
KeyVault__Endpoint: "<KeyVault_Url>"
SqlServer__SchemaOptions__AutomaticUpdatesEnabled: "true"
DataStore: "SqlServer"
```

I am not sure if this is a bug or if I am missing some configuration. 

I appreciate any help you can provide, thanks!

AB#98021

**Comment by akatakritos** [comment]
Possibly related: https://github.com/microsoft/fhir-server/issues/2923

**Comment by EXPEkesheth** [comment]
@marincon-sarmiento - The issue should be fixed with latest OSS release (3.3.18).

**Comment by EXPEkesheth** [comment]
Closing issue with comment above.
